src/trunk.jl

```diff
@@ -32,8 +33,10 @@ function trunk(::Val{:Newton},
     error("trunk should only be called for unconstrained problems. Try tron instead")
   end
 
+  wks = Dict{Symbol,Any}(:x => x)
```

Comment:
That's a lot of storage. For this to be practical, I think we should let the user choose to store x.

Reply:
These should be only pointers, so they should be light.

Reply:
Oh I see. Then 👍